src/psyclone/transformations.py

```diff
@@ -1754,19 +1754,7 @@ def apply(self, nodes, options=None):
         # Check whether we've been passed a list of nodes or just a
         # single node. If the latter then we create ourselves a
         # list containing just that node.
-        from psyclone.psyir.nodes import Node
-        if isinstance(nodes, list) and isinstance(nodes[0], Node):
-            node_list = nodes
-        elif isinstance(nodes, Node):
-            node_list = [nodes]
-        else:
-            arg_type = str(type(nodes))
-            raise TransformationError("Error in {0} transformation. "
-                                      "Argument must be a single Node in a "
-                                      "schedule or a list of Nodes in a "
-                                      "schedule but have been passed an "
-                                      "object of type: {1}".
-                                      format(self.name, arg_type))
+        node_list = self.get_node_list(nodes)
```

Comment:
Please remove the ":raises:" part of the docstring above now that this routine does not directly raise any exceptions.

Reply:
Done.